org.civicrm.reqangular/src/common/decorators/angular-date/datepicker-popup.js

```diff
@@ -16,22 +16,30 @@ define([], function () {
                  */
                 attrs.datepickerPopup = HR_settings.DATE_FORMAT;
 
-                scope.$watch('isOpen', function (value) {
+                /**
+                 * Calculates position of the datepicker popup.
+                 * @param value
+                 */
+                var adjustPopupPosition = function (value) {
```

Comment:
You could move this function definition outside the `compile` function, to make the latter cleaner and more readable
```js
var original_link = $delegate[0].link;

function adjustPopupPosition(value) {
    // ...
}

$delegate[0].compile = function ()  {
    return function (scope, element, attrs, ngModel) {
        attrs.datepickerPopup = HR_settings.DATE_FORMAT;
        
        scope.$watch('isOpen', adjustPopupPosition);
        original_link.apply(this, arguments);
    }
};

// ...
```